io/io/src/TFile.cxx

```diff
@@ -3939,13 +3938,19 @@ TFile *TFile::OpenFromCache(const char *name, Option_t *, const char *ftitle,
             // try to fetch the file (disable now the forced caching)
             Bool_t forcedcache = fgCacheFileForce;
             fgCacheFileForce = kFALSE;
-            if (need2copy && !TFile::Cp(name, cachefilepath)) {
-               ::Warning("TFile::OpenFromCache", "you want to read through a cache, but I "
-                         "cannot make a cache copy of %s - CACHEREAD disabled",
-                         cachefilepathbasedir.Data());
-               fgCacheFileForce = forcedcache;
-               if (fgOpenTimeout != 0)
-                  return nullptr;
+            if (need2copy) {
+               TUUID uuid;
```

Comment:
Since this is a transient name.  `gSystem->GetPID()` will be sufficient and **might** be faster than TUUID.

Reply:
done